spec/lib/submission_builder/ty2022/states/return_header_spec.rb

```diff
@@ -106,13 +111,15 @@
                 intake.direct_file_json_data.primary_filer.first_name = nil
                 intake.direct_file_json_data.primary_filer.middle_initial = nil
                 intake.direct_file_json_data.primary_filer.last_name = nil
+                intake.primary_suffix = primary_suffix
               end
 
               if intake.direct_file_json_data&.spouse_filer.present?
                 intake.direct_file_json_data.spouse_filer.dob = nil
                 intake.direct_file_json_data.spouse_filer.first_name = nil
                 intake.direct_file_json_data.spouse_filer.middle_initial = nil
                 intake.direct_file_json_data.spouse_filer.first_name = nil
+                intake.spouse_suffix = nil
```

Comment:
& same here does this expectation need to be added, since the test data is being set up: 
```
expect(doc.at('Filer Secondary TaxpayerName LastName')).not_to be_present
```